cppcache/integration-test/fw_helper.hpp

```diff
@@ -131,10 +131,7 @@ class TestException {
         m_filename(const_cast<char*>(filename)) {}
 
   void print() {
-    char buf[256];
-    apache::geode::client::Log::formatLogLine(
-        buf, apache::geode::client::LogLevel::Error);
-    fprintf(stdout, "--->%sTestException: %s in %s at line %d<---\n", buf,
+    fprintf(stdout, "--->%sTestException: %s in %s at line %d<---\n", apache::geode::client::Log::formatLogLine(apache::geode::client::LogLevel::Error).c_str(),
```

Comment:
Why is the logic of this method inverted? Before the Log::formatLogLine formatted the message, now it only formats an empty string? Very confused.

Reply:
formatLogLine used to take a char* buffer and do icky snprintf stuff, now it _returns_ a formatted std::string.  The name is kind of misleading now, maybe it should be something like getLogLinePrefix?